mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java

```diff
@@ -202,4 +203,38 @@ public static void updateObject(String objectIDString) {
         MCRPersistentIdentifierEventHandler.updateObject(object);
     }
 
+    @MCRCommand(syntax = "create pi with {0} for object {1} with additional ({2})",
+        help = "Creates a persistent identifier with the pi service with the id {0} for the object {1}"
+            + " with additional ({2}). Does nothing if the object already has a pi from the service {0}.",
+        order = 90)
+    public static void createPIForObjectIfNotExist(String serviceID, String objectIDString, String additional)
+        throws MCRAccessException, ExecutionException, MCRActiveLinkException, MCRPersistentIdentifierException,
+        InterruptedException {
+        final MCRObjectID objectID = MCRObjectID.getInstance(objectIDString);
+
+        if (!MCRMetadataManager.exists(objectID)) {
+            LOGGER.error("Object {} does not exist!", objectID);
+            return;
+        }
+
+        MCRPIService<MCRPersistentIdentifier> registrationService = MCRPIServiceManager
+            .getInstance().getRegistrationService(serviceID);
+
+        if (registrationService.isCreated(objectID, "")) {
+            LOGGER.info("Object {} already has a DOI!", objectID);
+        }
+        final MCRBase object = MCRMetadataManager.retrieve(objectID);
+        final MCRPersistentIdentifier doi = registrationService.register(object, additional);
+        LOGGER.info("Registered pi with {}: {} for object {}", serviceID, doi.asString(), objectID);
+    }
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 135).](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4254170)